### DISCUSSION TITLE
Github - Project

### DISCUSSION THREAD

**Discussion by robsonbittencourt** [frontendbr/forum#325] [Dicas]
Alguém está utilizando a nova feature do [Github chamada Project](https://www.youtube.com/watch?v=C6MGKHkNtxU)? 

Eu estou utilizando em alguns [projetos meus](https://github.com/hubot-js/hubot.js/projects/1?fullscreen=true) como forma de organizar as versões. 

Antes eu tinha um pouco de coisa no Trello, Keep, .txt. Transformei tudo em issues e estou usando o Project para gerenciar. 

Acho bem interessante para dar visibilidade do que está rolando no projeto. 

O que acham? Tem ideias de outros workflows que poderiam ser utilizados com a ferramenta?

**Comment by RicardoWEBSiTE** [comment]
Ainda não cheguei a usar essa nova feature porém tem o https://overv.io é
free e é inspirado no Trello.

Em 13 de outubro de 2016 22:07, Robson notifications@github.com escreveu:

**Comment by fititnt** [comment]
No nosso GT estamos usando em dois projetos. Bem bacana pra acompanhar projetos com muitos tópicos de forma mais visual sem depender de mais uma ferramenta externa como trello pra explicar pra equipe.

https://github.com/webiwg/acessibilidade-web-pt/projects/1

https://github.com/webiwg/historia-web-pt/projects/1

**Comment by RuanAragao** [comment]
Estamos utilizando no projeto Peba https://github.com/teresinahc/peba/projects/1
Eu achei muito bom o github ter incorporado essa feature, pois antes utilizava (e ainda utilizo em um projeto) o http://zenhub.io, que tem a função de incorporar tal utilidade na DOM do github.

**Comment by felipefialho** [comment]
Po, legal, funciona parecido com o a parte de to-do no Jira.

**Comment by danilovaz** [comment]
Começamos a usar no Mentoria, [aqui](https://github.com/training-center/training-center.github.io/projects/1). Particularmente, acho benéfico essa centralização ao invés de ficar usando aplicativos de terceiros. Facilita a comunicação e integração.

**Comment by giioohbernini** [comment]
Bem legal essa feature do github, manter tudo em um lugar facilita bastante. Porém, para projetos que tenham pessoas de outras áreas atuando nesse meio, o trello com o power-up do github ainda é mais benéfico na minha opinião (principalmente agora que da para colocar um power-up de graça :D ) !
